bridge-ui/src/hooks/useTokens.ts

```diff
@@ -13,19 +13,19 @@ const useTokens = (): Token[] => {
     if (fromChain.testnet) {
       if (fromChain.layer !== ChainLayer.L2) {
         return tokensList.SEPOLIA.filter(
-          (token) => !token.type.includes("native") && (config.isCCTPEnabled || token.symbol !== "USDC"),
+          (token) => !token.type.includes("native") && (config.isCctpEnabled || token.symbol !== "USDC"),
```

Comment:
"USDC" can be set as a constant `USDC_SYMBOL`

Reply:
done